### PR TITLE
Add EIP: SSZ execution blocks

### PR DESCRIPTION
**Pull request by etan-status**

### CODE REVIEW

File: EIPS/eip-####.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
```
Comment (abcoathup) [inline EIPS/eip-####.md:1]:
```suggestion
---
eip: 7807
```
Assigning next sequential EIP/ERC/RIP number..

Please also update the filename.

File: EIPS/eip-####.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+title: SSZ Execution blocks
+description: Migration of execution blocks to SSZ
+author: Etan Kissling (@etan-status), Gajinder Singh (@g11tech)
+discussions-to: https://ethereum-magicians.org/t/eip-ssz-execution-blocks/21580
```
Comment (abcoathup) [inline EIPS/eip-####.md:5]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/eip-7807-ssz-execution-blocks/21580
```
Added EIP number to Eth Magicians topic title

File: EIPS/eip-####.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
+---
+title: SSZ Execution blocks
```
Comment (abcoathup) [inline EIPS/eip-####.md:2]:
```suggestion
title: SSZ execution blocks
```
Does the E in execution need to be capitalized?

**Review by g11tech — APPROVED** [review]

**Review by cyberbossone-coder — APPROVED** [review]

**Review by perplexiphone-cmd — COMMENTED** [review]
Approve

**Pull request merged**